packages/perspective/src/js/perspective.js

```diff
@@ -541,12 +518,6 @@ view.prototype.to_json = async function(options) {
 
     let data;
 
-    if (options.format && options.format === "table") {
```

Comment:
removed this since it will break anyways, prev line 558 was always calling data.push which will fail if data was an object